task 3.txt

```diff
@@ -0,0 +1,5 @@
+solution :
+-file for managing the features and users (csv or yml)
```

Comment:
Where will the file be stored, how would it be maintained and by who?

Reply:
the file can be stored in source code repository (like Github) or it can be stored in S3 bucket.
it should be maintained by the devops team but in the most automatic way - for example in GitOps workflow, where changes to the infrastructure or configuration files trigger automated deployment pipelines.